src/java/src/main/java/org/wololo/flatgeobuf/Constants.java

```diff
@@ -8,8 +8,7 @@
 
     public static boolean isFlatgeobuf(ByteBuffer bb) {
         return bb.get() != MAGIC_BYTES[0] || bb.get() != MAGIC_BYTES[1] || bb.get() != MAGIC_BYTES[2]
```

Comment:
Can you verify this is correct?

I haven't tested the implementation but naively I'd assume this should be negated somewhere - something like:

```
let mismatch = bb.get() != MAGIC_BYTES[0] || bb.get() != MAGIC_BYTES[1] || bb.get() != MAGIC_BYTES[2] || bb.get() != MAGIC_BYTES[3];
return !mismatch;
```

Or maybe more directly:
`bb.get() == MAGIC_BYTES[0] && bb.get() == MAGIC_BYTES[1] && bb.get() == MAGIC_BYTES[2]`

Reply:
Nice catch - apparently I have used this function as the opposite of what it is named, i. here:

https://github.com/flatgeobuf/flatgeobuf/blob/6ca9b220123109e506ddb731373dcfc060e4b4c0/src/java/src/main/java/org/wololo/flatgeobuf/HeaderMeta.java#L76-L77

I'll fix it on master